01_install_requirements.sh

```diff
@@ -49,7 +49,7 @@ elif [[ $DISTRO == "rhel8" ]]; then
 
```

Comment:
Since https://github.com/metal3-io/metal3-dev-env/pull/912 merged, we'll need to update the pin on Line 22 above (we fix dev-scripts to a specific metal3-dev-env commit to avoid the risk of CI breakage in the event some upstream change breaks us)